src/components/notes/List/index.jsx

```diff
@@ -2,24 +2,34 @@
 import React from 'react'
 import Stack from 'cozy-ui/react/Stack'
 import { withBreakpoints } from 'cozy-ui/react'
-import AppTitle from './AppTitle'
+import { withClient } from 'cozy-client'
+import AppTitle from 'components/notes/List/AppTitle'
 import List from 'components/notes/List/List'
 import Add, { AddMobile } from 'components/notes/add'
+import useFetchNotesByIds from 'hooks/useFetchNotesByIds'
 
 import 'components/notes/List/list.styl'
 
-const ListView = ({ breakpoints: { isMobile } }) => {
+const shouldDisplayAddButton = (fetchStatus, notes) =>
+  fetchStatus === 'loaded' && notes.length > 0
```

Comment:
Nit: this could be done with useMemo.

Reply:
This operation is not heavy at all. I'm not sure about the use of a memo here